Add `ANZT6W` tournament page

JWWolstenholme opened the pull request:
<!-- 
  - Use [x] to complete the items
  - Remove the items unrelated to your work
  - Add any relevant information you consider useful
  - If there are no reviewers for your language, please mention it explicitly
-->

This PR adds an article for the [ANZT6W tournament](https://osu.ppy.sh/community/forums/topics/928661).

~~There have been a bunch of ANZT tournaments since this one that are undocumented in this wiki. I'm planning on documenting them all in one big pull request in the future. Let me know if that's not a good idea. They are:~~

This PR also adds entries on the tourney index page and ANZT tourney index page for all following tournaments after 6W. They are:
- [ANZT7S](https://osu.ppy.sh/community/forums/topics/1007139)
- [ANZT7W](https://osu.ppy.sh/community/forums/topics/1086888)
- [ANZT8S](https://osu.ppy.sh/community/forums/topics/1204722)
- [ANZT8W](https://osu.ppy.sh/community/forums/topics/1346048)
- [ANZT9S](https://osu.ppy.sh/community/forums/topics/1488646)
- [ANZT9W](https://osu.ppy.sh/community/forums/topics/1601049)
- [ANZT10S](https://osu.ppy.sh/community/forums/topics/1699251)
- [ANZT10W](https://osu.ppy.sh/community/forums/topics/1785325)
- [ANZT11S](https://osu.ppy.sh/community/forums/topics/1865121)
- [ANZT11W](https://osu.ppy.sh/community/forums/topics/1943093)
- [ANZT12S](https://osu.ppy.sh/community/forums/topics/2021371)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

Albionthegreat reviewed (changes requested):
When adding a tournament to the tournament listings in other languages, make sure to adjust the date formats. They're currently all using the English format.

I suggest adding all ANZT tournaments to the index page and tournament listings in this PR already. Once the full articles are PR'd, you then only need to add in the links. This will also avoid any merge conflicts that arise when doing multiple PR's for the same index page.

I will also suggest doing a new PR for each tournament, or PR's for each year (so bundle the summer and winter tournaments). Large PR's with tons of tournaments all at once are a daunting task to review.  By separating them into smaller chunks, they can be reviewed and taken care of 1 by 1. Divide and Conquer type of deal.

JWWolstenholme commented:
@Albionthegreat Thanks for the feedback. I've:
- Corrected the dates on the translated pages using the changes you suggest
- Added entries, but not dead links, to all the other ANZT tourneys since 6W.
  - I made sure to follow each translation's date format.
- Rebased this PR onto master since I've left this PR sitting for so long

I'll follow your advice and use smaller PRs for each tournament or two. Apologies for all the force pushes, hopefully that's not an issue.

JWWolstenholme commented:
I've also included links to tourneys 1 through 4 which were already included in the main tourney index page, but missing from the ANZT tourney index page. Forum links:

- [ANZT1](https://osu.ppy.sh/community/forums/topics/317172)
- [ANZT2](https://osu.ppy.sh/community/forums/topics/309684)
- [ANZT3](https://osu.ppy.sh/community/forums/topics/463813)
- [ANZT4](https://osu.ppy.sh/community/forums/topics/587374)

JWWolstenholme commented:
Just to clarify, this PR is ready for anyone to review it. I'm done making changes for now.

Albionthegreat reviewed (changes requested):
I'll take a look at the tournament page itself later. The listings and index pages look good!

JWWolstenholme commented on wiki/Tournaments/de.md:
```diff
@@ -149,13 +149,25 @@ Nicht offizielle Turniere/Wettkämpfe, die von den Communities veranstaltet werd
+| Aus/NZ osu!standard Tournament 10 - Winter | 23-07-2023 | 18-09-2023 | ::{ flag=AU }:: [cyo](https://osu.ppy.sh/users/8195473) | ::{ flag=AU }:: [Dumii](https://osu.ppy.sh/users/3068044) | ::{ flag=AU }:: [aknzx](https://osu.ppy.sh/users/9938943) |
+| Aus/NZ osu!standard Tournament 11 - Summer | 27-01-2024 | 11-03-2024 | santa deniers | jelqmaxxers | ASecretDozer |
+| Aus/NZ osu!standard Tournament 11 - Winter | 15-07-2024 | 08-09-2024 | ::{ flag=AU }:: [aknzx](https://osu.ppy.sh/users/9938943) | ::{ flag=AU }:: [cyo](https://osu.ppy.sh/users/8195473) | ::{ flag=AU }:: [Zyntex](https://osu.ppy.sh/users/11806306) |
+| Aus/NZ osu!standard Tournament 12 - Summer | 25-01-2025 | 23-03-2025 | *TBD* | *TBD* | *TBD* |
```
My bad! Good spot. Implemented.

Albionthegreat commented on wiki/Tournaments/ANZT/2019_Winter/en.md:
```diff
@@ -0,0 +1,314 @@
+
+### General rules
+
+- This is an osu!standard tourney for Australian/New Zealand players only. Participants must have an Australian or New Zealand flag on their osu profile.
```
```suggestion
- This is an osu! game mode tournament for Australian/New Zealand players only. Participants must have an Australian or New Zealand flag on their osu! profile.
```

Albionthegreat commented on wiki/Tournaments/ANZT/2019_Winter/en.md:
```diff
@@ -0,0 +1,314 @@
+### General rules
+
+- This is an osu!standard tourney for Australian/New Zealand players only. Participants must have an Australian or New Zealand flag on their osu profile.
+- Matches will be **Head to Head** with **Score V2**.
```
```suggestion
- Matches will be **Head-to-head** with **ScoreV2**.
```

Albionthegreat commented on wiki/Tournaments/ANZT/2019_Winter/en.md:
```diff
@@ -0,0 +1,314 @@
+| --: | :-- |
+| Round of 64 | Win four maps (Best-of-7) |
+| Round of 32 | Win five maps (Best-of-9) |
+| Round of 16, Losers Bracket Round 1 | Win five maps (Best-of-9) |
+| Quarterfinals, Losers Bracket Rounds 2 & 3 | Win six maps (Best-of-11) |
+| Semifinals, Losers Bracket Rounds 4 & 5 | Win six maps (Best-of-11) |
```
```suggestion
| Round of 16 | Win five maps (Best-of-9) |
| Quarterfinals | Win six maps (Best-of-11) |
| Semifinals | Win six maps (Best-of-11) |
```

Albionthegreat commented on wiki/Tournaments/ANZT/2019_Winter/en.md:
```diff
@@ -0,0 +1,314 @@
+- Players have up to ten minutes after their designated start time to move into the lobby. Failure to show will result in a forfeit. If both players are absent, the match will either be rescheduled or end in double forfeit to the discretion of the administration.
+- If a player disconnects within the first 30 seconds of a map, it will be aborted and replayed. Any disconnects afterwards will be a forfeit of that map.
+- Disconnects may only occur once per match per player. It will count as a forfeit after the first time.
+- Highest !roll has first warmup, ban and pick.
```
```suggestion
- Highest `!roll` has first warmup, ban and pick.
```

Albionthegreat commented on wiki/Tournaments/ANZT/2019_Winter/en.md:
```diff
@@ -0,0 +1,314 @@
+- Highest !roll has first warmup, ban and pick.
+- Each player is allowed one ban.
+- Each player is allowed one warmup. Warmups must not be longer than 5 minutes, must not be in any ANZT6W mappool and must be submitted.
+- One player may not pick from the same mod pool twice in a row (HD/HR/DT/FreeMod). This does not apply to NoMod.
+- In a FreeMod pick, each player **must select a mod(s)** (HD/HR/HDHR). **NoMod is not allowed**.
```
```suggestion
- One player may not pick from the same mod pool twice in a row (HD/HR/DT/Free Mod). This does not apply to No Mod.
- In a Free Mod pick, each player **must select a mod(s)** (HD/HR/HDHR). **No Mod is not allowed**.
```

Albionthegreat commented on wiki/Tournaments/ANZT/2019_Winter/en.md:
```diff
@@ -0,0 +1,314 @@
+- One player may not pick from the same mod pool twice in a row (HD/HR/DT/FreeMod). This does not apply to NoMod.
+- In a FreeMod pick, each player **must select a mod(s)** (HD/HR/HDHR). **NoMod is not allowed**.
+- Failed scores will only count if the player revives before the end of the map. If a player fails and their opponent does not, the point goes to the opponent. If both players fail, the point goes to the higher failed score.
+- If the match ends in a tie, players will play the tiebreaker. The tiebreaker will be played under FreeMod conditions, allowing NoMod.
```
```suggestion
- If the match ends in a tie, players will play the tiebreaker. The tiebreaker will be played under Free Mod conditions, allowing No Mod.
```

JWWolstenholme commented:
Thanks for the feedback @Albionthegreat. 

Looks like I'm out-dated on new styling since I was last here 6-7 years ago.

Albionthegreat reviewed (approved):
Looks good to me!
